src/utils/shadows-types.ts

```diff
@@ -0,0 +1,58 @@
+export interface Shadows {
```

Comment:
getting these from primitives would be preferred, but since we process them here that doesn't seem practical